lessons/Algebra2/exploring-exponential-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,163 @@
+
+Normally we think that "more data means more power!", and that the more data we include in our sample the more clearly we'll see any potential correlations. But in certain circumstances - like in our covid dataset! - the correlations in our sub-groups cancel each other out when we put the groups together! This is called Simpson's Paradox, and you can learn more at @link{https://en.wikipedia.org/wiki/Simpson%27s_paradox, Wikipedia}.
+
+Fortunately, we can use `filter` to split our dataset into multiple sub-groups, and then fit models to each sample.
+
+[.lesson-instruction]
+Create helper functions to filter at least three other states, then use them to define the covid tables for each state.
+
+=== Common Misconceptions
+
+It's extremely common for students to think that filtering a table *changes the original table*. This is NOT how it works in Pyret! Instead, the `filter` function always produces a _new_ table, containing only the Rows for which the supplied function evaluates to `true`
+
+=== Synthesize
+- Do you think the data for MA shows a linear relationship? Why or why not?
+- Do you think this data shows a quadratic relationship? Why or why not?
+- Do you think this data shows some other kind of relationship? Why or why not?
+- In what other situations would it be useful to filter a dataset?
+- Can you think of other examples where Simpson's Paradox might arise?
+
+== Fitting Exponential Models @duration{35 minutes}
+
+=== Overview
+
+Students are introduced to exponential models, and extend their sampling techniques to exponential relationships.
+
+=== Launch
+
+[.lesson-instruction]
+- Open @printable-exercise{graphing-models.adoc} and @online-exercise{https://www.desmos.com/calculator/09kyr4xlct, MA Covid Spread - Linear}.
+- Complete the *first* section on @printable-exercise{graphing-models.adoc}.
+- Using `lr-plot` in Pyret, come up with the best possible linear model, and write your answer at the bottom of @printable-exercise{exploring-covid.adoc}.
+- Class discussion: Are linear models a good fit for this data? Why or why not?
+
+Linear models capture _straight-line relationships_, where one quantity varies proportionally based on another. In linear models, we expect the response variable to grow by equal amounts over equal intervals in the explanatory variable.
+
+@right{@image{images/MA-covid.png}} If we make the line go from the start to the peak, almost all of the points bulge out below out line of best fit. If we make the line fit the bulge, all the points fall above it! We always have either too many points _below_ the line in the middle or _above_ the line at the end. **It's growing too fast to be fit with a linear model that grows at a constant rate!**
+
+[.lesson-instruction]
+- Take a few minutes to explore @online-exercise{https://www.desmos.com/calculator/73y2med4yj, MA Covid Spread - Quadratic}.
+- Complete the *second* section on @printable-exercise{graphing-models.adoc}.
```

Comment:
Too much going on in this lesson-instruction. I'd inlcude the first two bullets, then some discussion prompts. Feels overwhelming to have two different platforms in such a quick succession. 

There could be some text to offer teachers guidance on how to help kids make connections between desmos and Pyret and why we've chosen to include desmos before you jump back to Pyret.

(Come to think of it, teachers might like an explanation for the inclusion of Desmos when it first appears. Could disucss how differnet tools have different affordances, blah blah.)